### PR TITLE
Pet food

### PR DESCRIPTION
**Pull request by morecallan**
@Megraohoh - Is this ready for your sib to review?

### CODE REVIEW

**Comment by Megraohoh** [comment]
@alexsaber

File: index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<!DOCTYPE html>
```
Comment (alexsaber89) [inline index.html]: This line can be removed.

**Comment by alexsaber89** [comment]
@Megraohoh nice job with this!  👍

**Comment by Megraohoh** [comment]
Ready for merge?

**Comment by alexsaber89** [comment]
👍

**Pull request merged**